www/pfSense-pkg-squid/files/usr/local/pkg/squid.inc

```diff
@@ -788,6 +804,9 @@ function squid_validate_general($post, &$input_errors) {
 	unset($port, $webgui_port);
 
 	if ($post['transparent_proxy'] == 'on') {
+		if ($post['listenproto'] == 'inet6') {
+			$input_errors[] = "Transparent Proxy is not working for IPv6 only mode.";
```

Comment:
Reword this to `"Transparent Proxy does not support IPv6."`

Reply:
fixed